app/knight.py

```diff
@@ -0,0 +1,24 @@
+class Knight:
+    def __init__(self, name: str, power: int, hp: int) -> None:
+        self.name = name
+        self.power = power
+        self.hp = hp
+
+    def using_armour(self, armour: list) -> int:
+        for part in armour:
+            self.hp += part["protection"]
+        return self.hp
+
+    def using_weapon(self, weapon_power: int) -> int:
+        self.power += weapon_power
+        return self.power
+
+    def using_potion(self, potion: dict) -> callable:
```

Comment:
For the return type hint, use Tuple[int, int] instead of callable. Import Tuple from the typing module.

Reply:
can I do annotation without importing form typing module?